xapi-client/src/test/java/dev/learning/xapi/client/configuration/XapiClientConfiguration2Test.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2016rue-2023 Berry Cloud Ltd. All rights reserved.
+ */
+package dev.learning.xapi.client.configuration;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+
+import dev.learning.xapi.client.XapiClient;
+import dev.learning.xapi.client.configuration.XapiClientConfiguration2Test.XapiTestClientConfiguration2;
+import java.io.IOException;
+import okhttp3.mockwebserver.MockResponse;
+import okhttp3.mockwebserver.MockWebServer;
+import okhttp3.mockwebserver.RecordedRequest;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.web.reactive.function.client.WebClientAutoConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpHeaders;
+import org.springframework.web.reactive.function.client.WebClient.Builder;
+
+/**
+ * XapiClientConfiguration2 Test.
```

Comment:
Please replace 2 with a suitable name. XapiClientAutoConfiguration X Test (where x is what is being tested)